consensus-peer/local-deploy.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
```

Comment:
I generally like shell scripts, though feel like maybe our methods should be consolidated.  We have (at least) a mixture of make, shell, tcl, behave, and docker compose.